Products/PortalTransforms/Transform.py

```diff
@@ -176,6 +175,7 @@ def _load_transform(self):
         return transform
 
     security.declarePrivate('manage_beforeDelete')
+
```

Comment:
valid point. this was an outcome of autopep8. also the change ``implements`` -> ``@implementer`` is missing